src/js/timer.js

```diff
@@ -0,0 +1,107 @@
+import "../../node_modules/howler/src/howler.core.js";
```

Comment:
Еще одна прелесть вебпака это то что, он нам позволяет импортировать из commonJs модулей, как будто они es-модули. Поэтому теперь можно импортировать из Howler как из Luxon)
```suggestion
import { Howl } from "howler";
```